gio/src/subclass/list_model.rs

```diff
@@ -146,3 +147,25 @@ where
     };
     item.to_glib_full()
 }
+
+impl ListModel {
```

Comment:
You need to put this into `src/list_model.rs` and put it on some `ListModelExtManual` trait